inc/saiswitch.h

```diff
@@ -349,7 +349,14 @@ typedef enum _sai_switch_attr_t
     /** Nonvolatile storage required by both SAI and NPU in KB [sai_uint64_t]
      * Will be 0 for SAI_RESTART_TYPE_NONE */
     SAI_SWITCH_ATTR_NV_STORAGE_SIZE,
-
+    
+    /** Whether action list is mandatory for table creation [bool] 
+     * Default [false]*/
+    SAI_SWITCH_ATTR_ACL_TABLE_ACTION_LIST_MANDATORY,
+
+    /** List of actions supported [sai_u32_list_t] 
+     * List of all supported actions will be returned */
+    SAI_SWITCH_ATTR_ACL_TABLE_ACTION_LIST,
```

Comment:
What is use of SAI_SWITCH_ATTR_ACL_TABLE_ACTION_LIST_MANDATORY is this something that cant be knokced off while creating the tables. Also all ACL tables (in all stages) might not support same set of actions, so how will this global be useful. Do we assume mandatory to be same for all stages etc. 
Typically Keys can be programmed to save on the TCAM width. What is the use of restricting actions, could ASICs get advantage on the width of action memory (say SRAM etc)